docs/reference/how-to/size-your-shards.asciidoc

```diff
@@ -544,3 +544,37 @@ PUT _cluster/settings
   }
 }
 ----
+
+[discrete]
+==== Number of documents in the shard cannot exceed [2147483519]
+
+
+Elasticsearch shards reflect Lucene's underlying https://github.com/apache/lucene/issues/5176[index 
+`MAX_DOC` hard limit] of 2,147,483,519 (`(2^31)-129`) docs. This figure is
+the sum of `docs.count` plus `docs.deleted` as reported by the <<indices-stats,Index stats API>> 
+per shard. Exceeding this limit will result in errors like the following:
+
+[source, txt]
+----
+Elasticsearch exception [type=illegal_argument_exception, reason=Number of documents in the shard cannot exceed [2147483519]]
+----
+
+TIP: This calculation includes nested documents.
+This may differ from the results of the <<search-count,Count API>>.
+Therefore, it's results are not guaranteed to line up to the <<search-count,Count API>> 
+which masks nested docs. 
+
+
+Try using the <<indices-forcemerge,Force merge API>> to free up space and recover the
+ability to admin the index. For example:
```

Comment:
"to admin"? Not sure what you're trying to say with that. Again, I think this'll be confusing to folks for whom English is a foreign language.